packages/react/src/themes/teams/components/Icon/svg/icons/index.ts

```diff
@@ -73,6 +73,7 @@ import giphy from './giphy'
 import hand from './hand'
 import highlight from './highlight'
 import horizontalRule from './horizontalRule'
+import imageUnavailabe from './image-unavailable'
```

Comment:
should this be imageUnavailable instead? :)

Reply:
Yes!